app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,42 @@
+class MetricsController < ActionController::Base
```

Comment:
Is it safe for this route to have no auth?

Reply:
Added basic auth